src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState: [],
+  reducers: {
+    addBook: (state, action) => ({
+      ...state,
+      books: [...state.books, action.payload],
+    }),
+    removeBook: (state, action) => ({
+      ...state,
+      books: state.books.filter((book) => book.id !== action.payload.id),
+    }),
```

Comment:
- As I mentioned above, please use the `removeBook` action to mutate the state and not return a new object, please fix this 🙏

Reply:
ok I will fix it